internal/tiltfile/live_update.go

```diff
@@ -63,8 +63,13 @@ var _ starlark.Value = liveUpdateRunStep{}
 var _ liveUpdateStep = liveUpdateRunStep{}
 
 func (l liveUpdateRunStep) String() string {
-	return fmt.Sprintf("run step: %s", strconv.Quote(l.command))
+	s := fmt.Sprintf("run step: %s", strconv.Quote(l.command))
+	if len(l.triggers) > 0 {
```

Comment:
fwiw, I had left these out since I figured the command would probably suffice to identify the step to the user and the triggers might just feel like noise, but I don't really feel strongly either way

(and I should have at least commented that intent)

Reply:
ah gotcha. I'll leave them in for visibility purposes -- i.e. there's no other way for the user to check that their triggers got correctly populated.